mlflow/types/chat.py

```diff
@@ -62,6 +62,12 @@ def to_tool_call(self, id=None) -> ToolCall:
         return ToolCall(id=id, type="function", function=self)
 
 
+class FunctionCall(Function):
```

Comment:
since this is not a generic/broadly used dataclass, let's just keep this in `mlflow.genai.judges` somewhere